docs/app/pages/getting-started/getting-started.component.html

```diff
@@ -1,104 +1,132 @@
+        <li><a routerLink="." fragment="seed-project">Seed Project</a></li>
+        <li><a routerLink="." fragment="add-to-an-existing-project">Add to an Existing Project</a></li>
+        <li><a routerLink="." fragment="consuming-ux-aspects-components">Consuming UX Aspects Components</a></li>
+        <li><a routerLink="." fragment="contributing">Contributing</a></li>
+        <li><a routerLink="." fragment="getting-help">Getting Help</a></li>
+    </ol>
+
+    <section>
+        <h2 id="pre-requisites">Pre-Requisites</h2>
+
+        <p>Instructions on this page refer to the following software:</p>
+
+        <ul>
+            <li>
+                <a href="https://nodejs.org/">Node.js</a> v10 &mdash; this provides the <code>npm</code> command, which
+                is used to install application dependencies.
+            </li>
+            <li>
+                <a href="https://git-scm.com/">Git</a> &mdash; used to download the seed project, or to build UX Aspects
+                from source.
+            </li>
+            <li>
+                <a href="https://cli.angular.io/">Angular CLI</a> &mdash; used to build the seed project or create a new
+                Angular application.
+            </li>
+            <li><a href="https://gruntjs.com/using-the-cli">Grunt</a> &mdash; used to build UX Aspects from source.</li>
+        </ul>
+    </section>
+
+    <section>
+        <h2 id="seed-project">Seed Project</h2>
+
+        <p class="m-b">
+            The easiest way to get started with UX Aspects is to download our
+            <a href="https://github.com/UXAspects/UXAspects-Seed">Seed Project</a>. This is a starter project, suitable
+            for building a new Angular application using UX Aspects. Since it is built with Angular CLI, it is easy to
+            extend with new pages and components.
+        </p>
+
+        <ol>
+            <li>
+                Clone the repository:
+                <code>git clone https://github.com/UXAspects/UXAspects-Seed.git <i>my-project</i></code>
+            </li>
+            <li>Switch to the directory which was downloaded.</li>
+            <li>Install the dependencies: <code>npm install</code></li>
+            <li>Build and run the site using the Angular CLI development server: <code>ng serve</code></li>
+        </ol>
+
+        <p>
+            A complete guide on using the Seed Project can be
+            <a href="https://github.com/UXAspects/UXAspects-Seed/blob/master/README.md">found here</a>.
+        </p>
+    </section>
+
+    <section>
+        <h2 id="add-to-an-existing-project">Add to an Existing Project</h2>
+
+        <p>If you want to consume UX Aspects in an existing application follow the steps below:</p>
+
+        <h3>Install UX Aspects</h3>
+
+        <pre>npm install --save @ux-aspects/ux-aspects</pre>
+
+        <p>
+            Angular 6 applications should also install <code>rxjs-compat</code>.
```

Comment:
Just noticed this should also be changed. `rxjs-compat` is required all all versions of Angular we support (until we remove our dependency on `angular-split`

Reply:
Should it then be a dependency of the library, or a peer dependency?

Reply:
It would need to be a peer dependency as the `rxjs-compat` version would need to match the version of `rxjs` that is installed, which would vary based on the version of Angular that the application is using